app/controllers/players_controller.rb

```diff
@@ -34,8 +34,11 @@ def destroy
 
   def update
     @player = Player.find(params[:id])
+    @player.attributes = player_params
+    @player.suspend!
```

Comment:
extract data from `suspend` method and merge it to `player_params`, then just use normal `update`